src/main/java/com/eleks/academy/whoami/model/response/History.java

```diff
@@ -0,0 +1,25 @@
+package com.eleks.academy.whoami.model.response;
+
+import com.eleks.academy.whoami.core.action.PlayerAction;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class History {
+
+	private List<List<PlayerAction>> turns;
```

Comment:
easier just to return `List<List<PlayerAction>>` from contoller, no need of object that consists if only one response field